### PR TITLE
writing something personal

### PR DESCRIPTION
**Pull request by wutongy**
leave my part of adventure.

### CODE REVIEW

**Comment by wutongy** [comment]
Please review this, @casey-collab

File: Chinese/16fall_USA/Columbia_university/columbia I am coming.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+写于2016/4/13 比利时布鲁塞尔自由大学图书馆
+经历了一个不算满意的申请季，最后决定去哥大了。
+未来两年必须努力，争取能读david blei 或者collins 的phd.
+也算正式转码了。
+[这个世界做事选对方向也很重要啊！](../the-right-way/to_be_a_coder.md)
```
Comment (casey-collab) [inline Chinese/16fall_USA/Columbia_university/columbia I am coming.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Chinese/16fall_USA/Columbia_university/columbia I am coming.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+写于2016/4/13 比利时布鲁塞尔自由大学图书馆
```
Comment (casey-collab) [inline Chinese/16fall_USA/Columbia_university/columbia I am coming.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wutongy** [comment]
Please review this, @casey-collab

File: Chinese/大话西游.md [hunk 3]
```diff
@@ -104,4 +104,5 @@
 
 [ZERO](ZERO/ZERO.md)
 
-[小至尊宝](yixuanda/yixuanda.md)
\ No newline at end of file
+[小至尊宝](yixuanda/yixuanda.md)
+[美帝征程](16fall_USA/Columbia_university/columia I am coming.md)
```
Comment (casey-collab) [inline Chinese/大话西游.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline Chinese/大话西游.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: Chinese/16fall_USA/Columbia_university/columbia I am coming.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (casey-collab) [inline Chinese/16fall_USA/Columbia_university/columbia I am coming.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wutongy** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :palm_tree:

**Pull request merged**